utils/validate-data-source.js

```diff
@@ -1,13 +1,35 @@
 const appRoot = require('app-root-path');
+const config = require('config');
+const _ = require('lodash');
 
-const { validateFilePath } = appRoot.require('api/v1/db/json/fs-operations');
-
-const dbPath = 'tests/unit/mock-data.json';
+const { dataSources } = config.get('dataSources');
+const json = dataSources.includes('json')
+  ? appRoot.require('api/v1/db/json/fs-operations').validateJsonDb
+  : null;
+const oracledb = dataSources.includes('oracledb')
+  ? appRoot.require('api/v1/db/oracledb/connection').validateOracleDb
+  : null;
 
 /**
  * @summary Validate database configuration
  * @function
+ * @param {string} dataSourceType data source type
```

Comment:
Since it looks like the function doesn't take any parameters, should this get removed?

Reply:
Good catch, will fix.